.github/workflows/check-for-releases.yml

```diff
@@ -0,0 +1,24 @@
+name: Check for new Redmine releases
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run this Action every day a 5:35am UTC
+    - cron: "37 5 * * *"
+
+jobs:
+  redmine-release:
+    name: Check for Redmine updates
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check for updates
+        uses: git-for-windows/rss-to-issues@v0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          feed: https://www.redmine.org/projects/redmine/news.atom
+          prefix: "Update the Behat tests: "
+          #character-limit: 255
```

Comment:
needed?

Reply:
I've tested the limit with a dry-run, see https://github.com/Art4/php-redmine-api/actions/runs/9681576974/job/26712484988

The content of the issue would be:

```
Redmine [5.1.3](https://www.redmine.org/versions/195) and [5.0.9](https://www.redmine.org/versions/194) have been released and are now available for [download](https://www.redmine.org/projects/redmine/wiki/Download). These are maintenance releases and onl…

---
## Would you like to know more?
Read the full article on the following website:

https://www.redmine.org/news/145
```

I think the full content would be nicer, so I have commented out the limit.